lib/ui/widgets/patcherView/patch_selector_card.dart

```diff
@@ -1,3 +1,4 @@
+import 'package:collection/collection.dart';
```

Comment:
```suggestion
```

I don't think this package is related to sorting, please remove them or mark this as resolve.

👋 Anyway, thank for your contribution!